bin/test

```diff
@@ -1,7 +1,24 @@
 #!/bin/bash
-set -euo pipefail
+set -uo pipefail
 
-echo "--- :ruby: Bundle Install"
-bundle install --jobs 4 
-echo "+++ :kubernetes: Run Tests"
-bundle exec rake test TESTOPTS='-v'
\ No newline at end of file
+function print_header() {
```

Comment:
Here's what it looks like locally (CI remains the same):

![image](https://user-images.githubusercontent.com/4789493/35177517-0920fac4-fd4e-11e7-83ae-70c38cb3a423.png)